tests/cuda/test_ckf_toy_detector.cpp

```diff
@@ -249,8 +249,8 @@ INSTANTIATE_TEST_SUITE_P(
 INSTANTIATE_TEST_SUITE_P(
     CkfToyDetectorValidation1, CkfToyDetectorTests,
     ::testing::Values(std::make_tuple(
-        "toy_n_particles_10000", std::array<scalar, 3u>{0.f, 0.f, 0.f},
+        "toy_n_particles_1000", std::array<scalar, 3u>{0.f, 0.f, 0.f},
         std::array<scalar, 3u>{0.f, 0.f, 0.f},
         std::array<scalar, 2u>{1.f, 100.f}, std::array<scalar, 2u>{-3.f, 3.f},
         std::array<scalar, 2u>{0.f, 2.0f * detray::constant<scalar>::pi}, -1.f,
-        10000, 1)));
+        1000, 1)));
```

Comment:
:confused: Why did you reduce the number of particles? The other parameter changes I don't find too worrisome. (Maybe because I don't fully understand them.) But if we cannot reconstruct 10k particles anymore, that's a bit worrisome...

Reply:
Probably it is Ok with CI.
But my laptop (rtx 2070) does not pass the test due to OOM error, which I don't know why

Reply:
Regarding that this is not supposed to happen, the current CKF performance with GPU will be pretty bad compared to what I have shown before. 

We will definitely need to fix it soon, but let us focus on running ODD at the moment

Reply:
Hmm... OOM. Just what I like to hear... :frowning:

Fine, let's reduce the number of particles for now then.

Reply:
I increased the bin capacity in the grids, which will overallocate memory now. I had not expected this to cause an OOM for the toy detector though. I has only 18 grids...

Reply:
It might not be from detray v51 as CI works OK. Instead it could be triggered by other PRs in the past.

Reply:
Therefore, I think the only problem we have is the wire chamber navigation - but anyway it is not even working well in detray helix navigation so it is not very surprising...